DESCRIPTION

```diff
@@ -23,6 +23,7 @@ Imports:
     checkmate,
     dplyr,
     grDevices,
+    htmltools,
```

Comment:
This may be redundant as we `Depend` on `shiny` and `shiny` `Imports` `htmltools`.
Granted, `tagInsertChildren` is not re-exported by `shiny`.